core/src/main/java/io/parsingdata/metal/Shorthand.java

```diff
@@ -99,154 +100,215 @@
 
     private Shorthand() {}
 
+    /** Instantiates a {@link Def} nested in a {@link Post}. */
```

Comment:
To keep this similar with `nod`,`opt` and `con`: Maybe start with:

```
/** "DEFinition" : ...
```

Reply:
I've added them everywhere for consistency, even when the shorthand is not really an abbreviation.